src/NetSapiensSharp/Connector.cs

```diff
@@ -16,6 +16,7 @@ public class Connector : IDisposable
         private int _ExpiresInSeconds;
         private DateTime _ExpirationStartTime = DateTime.UtcNow;
         private const int _UnauthorizedRetryLimit = 5;
+        private const int _ErroredRetryLimit = 5;
```

Comment:
What is the upper time limit for a single try? Knowing that, we might determine that waiting 5 times for that limit is too high.

Can you give some details of how this might not be better solved by retrying this external to this library?